app/templates/main/userProfile.html

```diff
@@ -21,232 +21,268 @@ <h6 align="center">{{volunteer.email}}</h6>
+        </button>
+      </h3>
+      <div id="collapseFour" class="accordion-collapse collapse" aria-labelledby="headingFour" data-bs-parent="#userProfile">
+        <div class="accordion-body">
+          <!-- This makes sure that the CeltsAdmins can make someone program manager or not.  -->
+          {% if g.current_user.isCeltsAdmin %}
+            <h5>Which programs do you want to make {{volunteer.firstName}} {{volunteer.lastName}} manager for?</h5>
+          {% else %}
+            <h5>{{volunteer.firstName}} {{volunteer.lastName}} is the manager of:</h5>
+          {% endif %}
+          <ul class="list-unstyled">
+            {% for program in programs %}
+              {% set action = "add" %}
+              {% set checked = "" %}
+              {% if program.id in permissionPrograms %}
+                {% set action = "remove" %}
+                {% set checked = "checked" %}
+                {% if not g.current_user.isCeltsAdmin%}
+                  <li>{{program.programName}}</li>
+                {% endif %}
+              {% endif %}
+              {% if g.current_user.isCeltsAdmin%}
+                <li>
+                  <input  type="checkbox" id="mgr-{{ program.id }}" data-programid="{{program.id}}" {{checked}}  onclick="updateManagers(this,'{{volunteer.username}}')">
+                  <label  style="padding-left: 15px;" for="mgr-{{ program.id }}">{{program.programName}}</label>
+                </li>
+              {% endif %}
+            {% endfor %}
+           </ul>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+    <!-- End Program Manager section -->
+    <!-- Start Background Check Table -->
+    {% if g.current_user.isAdmin %}
+    <div class="accordion-item">
+      <h3 class="accordion-header" id="headingFive">
+        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
+            Background Check
+        </button>
+      </h3>
+      <div id="collapseFive" class="accordion-collapse collapse" aria-labelledby="headingFive" data-bs-parent="#userProfile">
+        <div class="accordion-body">
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th scope="col">Description</th>
+                   {% if g.current_user.isCeltsAdmin%}
+                     <th scope="col">Status</th>
+                     <th scope="col">Date</th>
+                     <th class="" id="displaySave" scope="col"></th>
+                   {% endif %}
+                 </tr>
+               </thead>
+               <tbody>
+                 {% for bgType in backgroundTypes %}
+                 <tr>
+                   <td>
+                       <h6>{{bgType.description}}</h6>
+                       <ul class="mb-0" id="bgHistory{{bgType.id}}" >
+                         {% if allBackgroundHistory[bgType.id][0] %}
```

Comment:
This conditional is unnecessary. the for loop will either happen or not anyway, which is what we want.